ffi/src/debug.rs

```diff
@@ -3,18 +3,43 @@ pub use self::implementation::*;
 #[cfg(feature = "debug_mode")]
 mod implementation {
     use std::fs::OpenOptions;
+    use std::path::PathBuf;
     use std::sync::Once;
 
     use tracing_subscriber::prelude::*;
     use tracing_subscriber::EnvFilter;
 
     static SETUP: Once = Once::new();
 
+    const SSPI_DBG_LOG_PATH_ENV: &str = "SSPI_DBG_LOG_PATH";
+
+    #[cfg(windows)]
+    const HOME_PATH_ENV: &str = "USERPROFILE";
+
+    #[cfg(not(windows))]
+    const HOME_PATH_ENV: &str = "HOME";
+
     pub fn setup_logger() {
         SETUP.call_once(|| {
-            let path = format!("{}/sspi-rs.log", std::env::var("HOMEPATH").unwrap());
+            let path = if let Ok(path) = std::env::var(SSPI_DBG_LOG_PATH_ENV) {
+                PathBuf::from(path)
+            } else if let Ok(path) = std::env::var(HOME_PATH_ENV) {
+                let mut path = PathBuf::from(path);
+                path.push("sspi-rs.log");
+                path
+            } else {
+                println!("[SSPI-DEBUG] Couldn’t find path for log file");
+                return;
+            };
```

Comment:
Here is the new logic to determine the log file.

```pwsh
$Env:SSPI_DBG_LOG_PATH_ENV = "C:\Users\david\sspi-rs.log"
```

Otherwise, it will try to write a `sspi-rs.log` file in the user’s home folder.